sys/linux/socket_unix.txt

```diff
@@ -48,14 +50,34 @@ sockaddr_un_abstract {
 	id	int32[20000:20004]
 }
 
-msghdr_un {
-	addr	ptr[in, sockaddr_un]
+send_msghdr_un {
+	addr	ptr[in, sockaddr_un, opt]
 	addrlen	len[addr, int32]
 	vec	ptr[in, array[iovec_in]]
 	vlen	len[vec, intptr]
 	ctrl	ptr[in, array[cmsghdr_un], opt]
 	ctrllen	bytesize[ctrl, intptr]
-	f	flags[send_flags, int32]
+	f	const[0, int32]
```

Comment:
[The source code](https://elixir.bootlin.com/linux/v5.13.6/source/net/unix/af_unix.c) actively uses flags, so the field ist definitely not obsolete and it's better to continue using `flags[send_flags, int32]` here instead of  `const[0, int32]`.

Reply:
Okay, will fix! I saw that the msghdr descriptions [here](https://github.com/google/syzkaller/blob/master/sys/linux/socket.txt#L144) set the field to 0, so maybe that should be changed as well.

It seems like msg_flags is used in `____sys_sendmsg()`, but AFAICT `allowed_msghdr_flags` is always 0 when called from the sendmsg syscall. Which means that the flags are never passed to the unix socket functions.

Reply:
Hmmm, yes, you're right. It's zeroed for all `sendmsg` calls, only seems to be nonzero when called from `sendmmsg` ([link](https://elixir.bootlin.com/linux/v5.13.2/source/net/socket.c#L2470))